### PR TITLE
Profile page without password change

### PR DESCRIPTION
**Pull request by JonasEbbinghaus25**

### CODE REVIEW

**Review by denniskp — APPROVED** [review]

File: src/main/web/src/scenes/Profile/index.tsx [hunk 1]
```diff
@@ -1,52 +1,133 @@
...
-import {useDetectAdBlock} from "adblock-detect-react";
-import {usePreventScrolling} from "../../organisms/ad-block-overlay/preventScrolling";
+import { useDetectAdBlock } from "adblock-detect-react";
+import { usePreventScrolling } from "../../organisms/ad-block-overlay/preventScrolling";
+import {fetchUserData, updatePassword} from "../../services/ProfileDataService";
+import { fetchUserImage } from "../../services/ProfilePictureService";
+import { updateAge, updateDescription, updateCourse, updateEmail, updateUsername, updatePicture } from "../../services/ProfileDataService";
 
-interface ProfileData {
+interface UserData {
```
Comment (maxschwinghammer) [inline src/main/web/src/scenes/Profile/index.tsx:16]: Best Practise: Extract this to a seperate file called `UserData` in a folder named `models`

File: src/main/web/src/services/ProfileDataService.tsx [hunk 2]
```diff
@@ -1,44 +1,171 @@
...
 
-const ProfileDataService = () => {
-    const [userData, setUserData] = useState({
-        username: null,
-        courseName: null,
-        age: null,
-        email: null,
-        imageData: null
-    });
+interface UserData {
```
Comment (maxschwinghammer) [inline src/main/web/src/services/ProfileDataService.tsx:4]: Use the model (see comment above)

**Review by maxschwinghammer — CHANGES_REQUESTED** [review]
Fixed

**Review by denniskp — APPROVED** [review]

**Review by FurKay00 — APPROVED** [review]

**Review by maxschwinghammer — CHANGES_REQUESTED** [review]
Delete the `change password` or implement it and extract the `UserModel`.
Then you can merge the PR.

**Review by Nahlam4 — APPROVED** [review]

**Review by maxschwinghammer — APPROVED** [review]

**Pull request merged**